src/libs/actions/User.js

```diff
@@ -0,0 +1,12 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+
+function getBetas() {
```

Comment:
Hmm, I feel like either this should go in PersonalDetails, or we should rename PersonalDetails to User and add it there instead of making two separate files. What do you think?

Reply:
I'm fine with two separate files. `PersonalDetails` is a bit different because it deals with the details of multiple people, whereas having `User` will restrict the functionality to things only being done for a single user.